js/blogs_info/blogs_info.js

```diff
@@ -1,6 +1,64 @@
 var blogs = [];
 
 /* 
-To add a new blog, add a new line in this format: "blogs.push(new blog('blog name', 'blog imageLocation', 'blog_link');"
+To add a new blog, add a new item to blogsInfo and push that item to blogs.
 */
-blogs.push(new blog("Gav's Dev Blog", "./images/blog_images/gavs_dev_blog.jpg", "https://gavsdevblog.wordpress.com/"));
+
+var blogsInfo = {
+  'loriansBlog': {
+    'name': "Lorian Bates",
+    'imageLocation': '',
+    'link': 'https://locodesblog.wordpress.com/2016/08/04/polymorphic-me/'
+  },
+  'mckinnelysBlog': {
+    'name': "McKinnely Bentley",
+    'imageLocation': '',
+    'link': 'http://mbcodeguy.blogspot.com/'
+  },
+  'matthewsBlog': {
+    'name': "Matthew David",
+    'imageLocation': '',
+    'link': 'https://mattattaqdevblog.wordpress.com/'
+  },
+  'gwensBlog': {
+    'name': "Gwendolyn Faraday",
+    'imageLocation': '',
+    'link': 'http://gwenfaraday.com/LearningCode/'
+  },
+  'nicholasBlog': {
+    'name': "Nicholas Gillespie",
+    'imageLocation': '',
+    'link': 'http://programpractical.com/'
+  },
+  'gavynsBlog': {
+    'name': "Gavyn Leavitt",
+    'imageLocation': './images/blog_images/gavs_dev_blog.jpg',
+    'link': 'https://gavsdevblog.wordpress.com/'
+  },
+  'jtsBlog': {
+    'name': "J.T. Maupin",
+    'imageLocation': '',
+    'link': 'https://jmsmaupin.wordpress.com/'
+  },
+  'kimberlysBlog': {
+    'name': "Kimberly Sharpe",
+    'imageLocation': '',
+    'link': 'http://kimmihack.blogspot.com/'
+  },
+  'larrysBlog': {
+    'name': "Larry Tooley",
+    'imageLocation': '',
+    'link': 'http://damagedtechnologies.com'
+  }
+};
+
+blogs.push(new blog(blogsInfo.loriansBlog));
+blogs.push(new blog(blogsInfo.mckinnelysBlog));
+blogs.push(new blog(blogsInfo.matthewsBlog));
+blogs.push(new blog(blogsInfo.gwensBlog));
+blogs.push(new blog(blogsInfo.nicholasBlog));
+blogs.push(new blog(blogsInfo.gavynsBlog));
+blogs.push(new blog(blogsInfo.jtsBlog));
+blogs.push(new blog(blogsInfo.kimberlysBlog));
+blogs.push(new blog(blogsInfo.larrysBlog));
+
```

Comment:
It may make more sense to make `blogsInfo` an array of objects that is looped though, and each object is pushed. That way you only need to update the object in the array, and not also update the `blogs.push`.